Misc/NEWS.d/next/Library/2020-05-23-00-22-11.bpo-40737.iph-CM.rst

```diff
@@ -0,0 +1 @@
+Handle PyModule_AddObject() errors in sqlite3 module init correctly.
```

Comment:
```suggestion
Fix possible reference leak for :mod:`sqlite3` initialization.
```

Reply:
Ah, neat with the .rst syntax. I'll make a note of that for future commits.